routes/userRoutes.js

```diff
@@ -27,14 +27,59 @@ routes.get('/', async (req, res, next) => {
  * @queryString : none,
 */
 routes.get('/:id', async (req, res, next) => {
-  const { id } = req.params;
   try {
-    const user = await models.User.where({ id });
-    if (user) {
-      res.status(200).json(user);
-    } else {
-      res.status(404).json(error.getUserById);
-    }
+    const user = await models.User.findById(req.params.id).exec();
+    res.send(user);
+  } catch (error) {
+    next(error);
+  }
+});
+
+/**
+ * [POST] Creates a new user
+ * @params : none,
+ * @body : none,
+ * @queryString : none,
+ */
+routes.post('/', async (req, res, next) => {
+  const user = req.body;
+  const newUser = models.User(user);
```

Comment:
Please add following steps:
- check if the `user.name` already exists in DB, if yes, return error message
- hash the `user.password` with bcrypt
- before sending the `result` object back in response, please remove the `password` property from it